taichi/runtime/llvm/runtime.cpp

```diff
@@ -1040,6 +1040,14 @@ int32 cuda_ballot_sync(int32 mask, bool bit) {
   return 0;
 }
 
+int32 cuda_ballot_i32(int32 predicate) {
+  return cuda_ballot_sync(UINT32_MAX, (bool)predicate);
+}
+
+int32 cuda_ballot_sync_i32(u32 mask, int32 predicate) {
```

Comment:
do we need and use this function?

Reply:
not yet,  in #4631 it has not yet been decided whether to block the mask or not, so both of them are temporarily implemented